### PR TITLE
Update to Paraiba parser

### PR DESCRIPTION
**Pull request by AnyaLindstromBattle**
csv.reader changed to csv.DictReader as there seemed to be an issue during data ingestion of columns not corresponding to their  respective indexes.

### CODE REVIEW

File: ingestion/functions/parsing/brazil_paraiba/paraiba.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+_DATE_DEATH_INDEX = "Data do Óbito"
```
Comment (attwad) [inline ingestion/functions/parsing/brazil_paraiba/paraiba.py:18]: nit but those are not "indexes" anymore, they are column names so please rename

File: ingestion/functions/parsing/brazil_paraiba/sample_data.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+M�todo,Munic�pio de Resid�ncia,Inicio Sintomas,Sexo,Doen�as preexistentes,Idade,Data do �bito
```
Comment (attwad) [inline ingestion/functions/parsing/brazil_paraiba/sample_data.csv:1]: is github messing with the encoding? These should appear fine if they are utf-8 encoded.

**Review by attwad — COMMENTED** [review]
There seems to be a bunch of files that should't be in this PR (template ,init, requirements etc), please make sure you ran:

```
git checkout main
git pull
git checkout paraiba
git merge main
git push
```

**Comment by AnyaLindstromBattle** [comment]
> There seems to be a bunch of files that should't be in this PR (template ,init, requirements etc), please make sure you ran:
> 
> ```
> git checkout main
> git pull
> git checkout paraiba
> git merge main
> git push
> ```

Yes, I noticed that. If I do that will it remove those excess files?

File: ingestion/functions/parsing/brazil_paraiba/paraiba.py (hunk 1 above)
Comment (AnyaLindstromBattle) [reply ingestion/functions/parsing/brazil_paraiba/paraiba.py:18]: will change

**Comment by attwad** [comment]
> > There seems to be a bunch of files that should't be in this PR (template ,init, requirements etc), please make sure you ran:
> > ```
> > git checkout main
> > git pull
> > git checkout paraiba
> > git merge main
> > git push
> > ```
> 
> Yes, I noticed that. If I do that will it remove those excess files?

One shall try and see, otherwise you can also try creating a new branch `git checkout -b fixparaiba` and push this new branch instead

File: ingestion/functions/parsing/brazil_paraiba/sample_data.csv (hunk 2 above)
Comment (AnyaLindstromBattle) [reply ingestion/functions/parsing/brazil_paraiba/sample_data.csv]: I really don't know what it going on with the encoding, it seems like either there is an encoding error or there is a KeyError where the key in paraiba.py is not being matched to the column in sample_data.csv

Comment (attwad) [reply ingestion/functions/parsing/brazil_paraiba/sample_data.csv]: In vscode you can check the encoding at the bottom right corner of the window (here UTF-8 on my screenshot below), if you click here you can save the file as the desired encoding, please try that

image

**Comment by AnyaLindstromBattle** [comment]
> omggg 🔥 🙇‍♂️ 👍

Hopefully it now ingests correctly too... could you please merge when you get the chance?

**Comment by attwad** [comment]
Yup, checks passed, merging now! thanks again

**Pull request merged**